doc/sphinxext/sphinx_gallery/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+==============
+Sphinx Gallery
+==============
+
+"""
+import os
+__version__ = '0.1.4'
```

Comment:
I would advise to use a more recent version of sphinx gallery. I think 0.1.6 is available at the moment. There is a script to download the latest sphinx-gallery:sphinx-gallery/bin/copy_sphinxgallery.sh.